server/models/GameLogic.js

```diff
@@ -84,7 +84,7 @@ class GameLogic {
         } else if (this.players.some(player => player.username === username)) {
             throw Error("You have already joined this game");
         } else {
-            const cards = cardsManager.assign(this.players, 5);
+            const cards = cardsManager.assign(this.players, 4);
```

Comment:
My review apparently didn't get submitted straight away.

Reply:
We changed the css so that cards don't squish to long rectangles, but then also until we make a real fix for them not just going off the page, I thought reducing the number of cards in your hand from 5 to 4 would do. I guess this will be a job for the UX sprint next week

Reply:
Was this the problem of it going of the bottom of the screen?

Reply:
No the cards weren't card shaped anymore, they were like tall rectangles, so I set `flex-shrink: 0` and changed the height so they wouldn't do that, but then we had the problem of them being a constant size and if there were too many cards they just went off the page horizontally so we just got rid of one of the cards as a quick fix

Reply:
As long as you think it won't have too much of an impact on game play. I'm a little hesitant but as long as it gets fixed in the next release...

Reply:
In the last release we had 3 cards in a hand, so going up to 4 is still an improvement, and I'd rather have less cards than cards that don't look like cards